plugin/type_uuid/item_uuidfunc.h

```diff
@@ -64,4 +64,48 @@ class Item_func_uuid: public Item_func_sys_guid
   { return get_item_copy<Item_func_uuid>(thd, this); }
 };
 
+class Item_func_sys_guid_v7: public Item_str_func
+{
+protected:
+  bool with_dashes;
+  size_t uuid_len() const
+  { return MY_UUID_BARE_STRING_LENGTH + with_dashes*MY_UUID_SEPARATORS; }
+public:
+  Item_func_sys_guid_v7(THD *thd): Item_str_func(thd), with_dashes(false) {}
+  bool fix_length_and_dec(THD *thd) override
+  {
+    collation.set(DTCollation_numeric());
+    fix_char_length(uuid_len());
+    return FALSE;
+  }
+  bool const_item() const override { return false; }
+  table_map used_tables() const override { return RAND_TABLE_BIT; }
+  LEX_CSTRING func_name_cstring() const override
+  {
+    static LEX_CSTRING name= {STRING_WITH_LEN("sys_guidv7") };
```

Comment:
I'm wondering why add `sys_guidv7` is the compatible with something else? `sys_guid` afaik is an oracle function so we just provided the equivalent. I'm not seeing a good case for including the alias for `sys_guidv7`.